.github/workflows/main.yml

```
@@ -7,6 +7,7 @@ on:
 
 env:
   DOTNET_VERSION: 9.0.x
+  COVERLET_EXCLUDE_COVERAGE: /p:Exclude=\"[Neo.UnitTests]*\"
```

```suggestion
  COVERLET_EXCLUDE_COVERAGE: -p:Exclude="[Neo.UnitTests]*"
```

to make it consistent with other files.

---

Now it fault

---

Its just a bug in the new version `COVERAGE` library, since we updated it.